kratos/utilities/intersection_utilities.h

```diff
@@ -38,6 +38,19 @@ namespace Kratos
 ///@name  Enum's
 ///@{
 
+enum class IntersectionUtilitiesLineIntersection
```

Comment:
```suggestion
enum class IntersectionUtilitiesTetrahedraLineIntersectionStatus
```
Besides I don't really love the idea of defining a new enum in the entire scope of Kratos. I'd do as we did so far, return an integer with the intersection status and properly document it.

Reply:
Note the comment below. This is not resolved.

Reply:
Before I did it with an integer, but then it was complex to understand the code when reading it. Can we pass with the enum?

Reply:
Hello @rubenzorrilla

Reply:
I think that if it is properly documented it is not very much difficult to understand the code. Besides it would be much less verbose, consistent with the other intersections and we avoid having a hanging enum in the complete Kratos namespace (I don't want misuses of this). I'm of course considering that, in my opinion, this is a pretty internal function that would potentially be used but very few developers.

I'd only be in favor of the enum if we agreed a standard intersection status one valid for all intersections. Not sure if possible.

Reply:
Okay, I will do something that may satisfy everyone